backend/wikimedia/wikicrawl.py

```diff
@@ -0,0 +1,45 @@
+# flake8: noqa W191
+import requests
+from bs4 import BeautifulSoup
+
+
+def wikimedia_crawl(page=100):
+	"""To get images from wikimedia when search for skin cancer.
+
+	Parameters
+		page : int.
+
+	Returns:
+		A list of dictionaries.
+		Each dictionary has
+			- Image source: an url
+			- Host: "Wikimedia"
+			- Original URL: an url
+			- Title: "Wikimedia <number>"
```

Comment:
![image](https://user-images.githubusercontent.com/19296926/99965614-abdc1080-2dc7-11eb-9a09-88e074d9a44a.png)
Can you extract these names instead?

Reply:
Similarly, the original URL should leads to [its own page](https://commons.wikimedia.org/wiki/File:Superficial_basal_cell_carcinoma.jpg)

Reply:
And the image URL should be get from that link, which is of original size instead of thumbnail.